#Service to clear the trajectory buffer on the robot

---
int64 ret
string msg
